aries-site/src/pages/foundation/accessibility.mdx

```diff
@@ -4,8 +4,13 @@ import {
   VideoSection 
 } from '../../examples/foundation';
 
+import { AccessVideo } from '../../components/accessibility';
+
 At HPE, we believe in being a force for good. We solve for humanity with humanity. We use technology to make the world better. We are unconditionally inclusive. **We believe that being accessible is a requirement, not an option**.
 
+In the video below, Bill Tipton, an HPE Product Accessibility Champion, discusses his role and the importance of inclusive design and development. The video shows Bill, a white man with blondish brown hair and a mustache, in his office wearing a long-sleeved lavender-blue dress shirt with the HPE rectangular-shaped, multi-color logo against a black background as the background.
```

Comment:
The second sentence seems like it belongs in an aria-describedby and `<div />` added as a child of `<video />`. Is there a reason it should be in the main content flow?

Reference: https://www.w3.org/WAI/PF/HTML/wiki/Media_Alt_Technologies#2:_Use_.40aria-describedby_for_the_longer_text_alternative

Reply:
bill and i had discussed this a way to provide an audio description for users but i think i could incorporate it with the aria-describedby as you mentioned and see what bill's experience is with that! that way it isn't unnecessary content on the page